nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/operations-contrib.xml

```diff
@@ -24,5 +24,11 @@
       </operation>
     </chain>
 
+    <chain id="Image.Blob.ConvertToPDF">
+      <operation id="Blob.RunConverter">
+        <param name="convert" type="string">pictureConvertToPDF</param>
```

Comment:
Shouldn't the parameter be `converter` here?
https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/blob/RunConverter.java#L51